app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

```diff
@@ -57,7 +57,16 @@ You could include:
 
 Teacher training providers want to see your passion and that you understand the bigger picture of teaching.
 
-Make sure you check your spelling and grammar in your application. You want to make the best possible impression.
+### How to write your personal statement
+
+When writing your personal statement you should make sure you check your spelling and grammar in your application. You want to make the best possible impression.
+
+You can use ChatGPT or other artificial intelligence (AI) tools to help you write your personal statement. You should not rely on it to write your entire statement because:
+
+* AI tools use bland language and will not be able to give details about you as a person and using them may result in your application being unsuccessful
+
+* your account to apply for teacher training may be blocked if you consistently submit personal statements that look like they have been written with AI tools
```

Comment:
I still think this needs to reinforce if you use them to write the whole thing for you  ...that look like they have been written **entirely by** AI tools or something along those lines.

Reply:
I've left it as the wording from apply, rose was unclear on the definition behind it and seemed to think they'd been decided as that